### DISCUSSION TITLE
New connector: Source Doubleverify

### DISCUSSION THREAD

**Discussion by iberchid** [airbytehq/airbyte#36130] [New Connector Request]
### Connector Name

DoubleVerify

### What type of integration

Source

### Revelant Information

DoubleVerify is the an independent platform for digital media measurement software and analytics that authenticates the quality and effectiveness of digital media for brands and media platforms.

The syncs would be run on a daily basis.

### Contribute

- [X] Yes, I want to contribute

**Comment by jrolom** [comment]
Hey @iberchid ! Glad to hear you are willing to contribute :) Have you developed the connector using the guidelines defined [here](https://docs.airbyte.com/connector-development/)?

**Comment by btkcodedev** [comment]
@jrolom I would like to implement the connector with low code

**Comment by btkcodedev** [comment]
Update: Connector-template generator is currently broken.
Waiting for it to be fixed

**Comment by btkcodedev** [comment]
Hi @iberchid Could you share your config token of Double Verify in order to complete this issue, It would be very helpful. My Airbyte Slack ID - btkcodedev
I asked DoubleVerify couple of times, but no response regarding the token it seems :(
We could share the credits for developing this source in the docs of DoubleVerify

**Comment by iberchid** [comment]
Hi @btkcodedev , the token I have is one of our clients' production account, therefore I cannot share it.

**Comment by btkcodedev** [comment]
Oh okay thanks!

[a comment by an automated account is not shown]
